cmd/skupper/skupper.go

```diff
@@ -82,68 +82,62 @@ func requiredArg(name string) func(*cobra.Command, []string) error {
 	}
 }
 
-func exposeTarget() func(*cobra.Command, []string) error {
```

Comment:
The diff in this file show a lot of changes, but actually there are just 4 or 5 lines modified moreover the new stringSliceContains method